automatic/palemoon/tools/chocolateyInstall.ps1

```diff
@@ -4,5 +4,16 @@ $url = '{{DownloadUrl}}'
 $url64 = '{{DownloadUrlx64}}'
 $silentArgs = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
 $validExitCodes = @(0)
+$pwd = "$(split-path -parent $MyInvocation.MyCommand.Definition)"
+
```

Comment:
Fair enough.  My particular preference goes the other way :-)  I am used to StyleCop telling me in my C# code if I have multiple lines between code segments.  This is not a major thing here though, just wanted to make sure I wasn't missing anything.